model/event/subscriptions_test.go

```diff
@@ -110,6 +111,24 @@ func (s *subscriptionsSuite) SetupTest() {
 			Owner:     "me",
 			OwnerType: OwnerTypePerson,
 		},
+		{
+			ID:      bson.NewObjectId(),
+			Type:    "type2",
+			Trigger: "trigger2",
+			Selectors: []Selector{
+				{
+					Type: "data",
+					Data: "somethingspecial",
+				},
+			},
+			RegexSelectors: []Selector{},
+			Subscriber: Subscriber{
+				Type:   SlackSubscriberType,
+				Target: &t5,
+			},
+			Owner:     "myProject",
```

Comment:
actually, use the same owner "me". This lets the test detect logic errors.

Reply:
done